### DISCUSSION TITLE
about import wiki data to postgresql

### DISCUSSION THREAD

**Discussion by tranquocnhat** [osm-search/Nominatim#3008] [General]
i have error when try import wiki data to postgresql
<img width="854" alt="Screen Shot 2023-03-12 at 16 00 06" src="https://user-images.githubusercontent.com/23132487/224534752-c9dda0cf-31c3-4a41-b817-07635ceaa683.png">
and there are my config:
- enviroment
<img width="652" alt="Screen Shot 2023-03-12 at 16 00 41" src="https://user-images.githubusercontent.com/23132487/224534786-0345056f-e5b7-4262-9ee9-79d4d73ed936.png">

**Comment by lonvia** [comment]
See #2948. This is fixed in master but not yet in a release.
